source/dotnet/Library/AdaptiveCards.Rendering.Wpf.Xceed/XceedTimeInput.cs

```diff
@@ -32,14 +33,16 @@ public static FrameworkElement Render(AdaptiveTimeInput input, AdaptiveRenderCon
                     timePicker.Value = value;
                 TimeSpan minValue;
                 if (IsSupportedTimeFormat(input.Min) && TimeSpan.TryParse(input.Min, out minValue))
-                    timePicker.EndTime = minValue;
+                    timePicker.StartTime = minValue;
```

Comment:
can you explain this change a bit on removing the default EndTime initialization here?